test/test_payment_initiation.rb

```diff
@@ -38,8 +38,10 @@ def test_all_routes
     create_payment_response = client.payment_initiation.create_payment(
       recipient_id,
       'testpayment',
-      currency: 'GBP',
-      value:    100.00
+      {
+        currency: 'GBP',
+        value:    100.00
+      }
```

Comment:
Needed to update this to fix the tests.

Reply:
oh man, how did this ever pass before?

Reply:
Looks like tests only failed because I was using ruby 3 locally (instead of ruby 2 which the repo uses) 😅  https://github.com/plaid/plaid-ruby/pull/332/commits/640899ef7aa1bdc8160f0739ac71ca7cb4457cb8